sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -151,9 +144,10 @@ case class RowDataSourceScanExec(
 
     Map(
       "ReadSchema" -> requiredSchema.catalogString,
-      "PushedFilters" -> seqToString(markedFilters.toSeq),
-      "PushedAggregates" -> aggString,
-      "PushedGroupby" -> groupByString) ++
+      "PushedFilters" -> seqToString(markedFilters.toSeq)) ++
+      pushedDownOperators.aggregation.fold(Map[String, String]()) { v =>
+        Map("PushedAggregates" -> seqToString(v.aggregateExpressions),
+          "PushedGroupby" -> seqToString(v.groupByColumns))} ++
```

Comment:
@sunchao had a nit comment to rename to `PushedGroupBy` in https://github.com/apache/spark/pull/34498#discussion_r744076628 . Though I feel `PushedGroupByColumns` are better. To me, it seems to mean the same thing between `aggregate` and `group by`.

Reply:
Changed to `PushedGroupByColumns`